<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>Using EMPOP for mitotype searches - EMPOP Documentation</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "Using EMPOP for mitotype searches";
        var mkdocs_page_input_path = "using_empop.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> EMPOP Documentation
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../revisions/">Revision Overview</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../introduction/">Introduction</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../concept/">Concept</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../register/">Register & Login</a>
                </li>
              </ul>
              <ul class="current">
                <li class="toctree-l1 current"><a class="reference internal current" href="#">Using EMPOP for mitotype searches</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#41-query-options">4.1 Query Options</a>
        <ul>
    <li class="toctree-l3"><a class="reference internal" href="#411-sample-id">4.1.1. Sample ID</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#412-ranges">4.1.2. Ranges</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#413-mitotype">4.1.3. Mitotype</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#414-release">4.1.4. Release</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#415-find-neighbors">4.1.5. Find neighbors</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#416-match-type">4.1.6. Match type</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#417-disregard-indels">4.1.7. Disregard InDels</a>
    </li>
        </ul>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#42-result">4.2. Result</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#43-details">4.3. Details</a>
        <ul>
    <li class="toctree-l3"><a class="reference internal" href="#431-when-matches-are-found">4.3.1. When matches are found</a>
    </li>
        </ul>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#44-neighbors">4.4. Neighbors</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#45-alignment">4.5. Alignment</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#46-haplogrouping">4.6. Haplogrouping</a>
    </li>
    </ul>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../browsing_empop/">Browsing EMPOP for populations</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../tools/">EMPOP Tools</a>
                </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">EMPOP Documentation</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
      <li class="breadcrumb-item active">Using EMPOP for mitotype searches</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h2 id="4-using-empop-for-mitotype-searches">4. Using EMPOP for mitotype searches</h2>
<p>EMPOP follows the revised and extended guidelines for mitochondrial DNA typing issued by the DNA commission of the ISFG (Parson et al. 2014). See document for further details.</p>
<p><strong>EMPOP’s query engine - SAM 2</strong>
EMPOP uses SAM 2, an updated and optimized software based on tests with carefully curated full mitogenome sequences to perform unbiased and conservative database queries to assist statistical evaluation of the evidence in forensic practice.
The major changes to the earlier version of the software SAM (Röck et al 2011) include:</p>
<ol>
<li>updated alignment/nomenclature conventions for the phylogenetically instable regions 50-70, 310-316, 455-460, 961-966, 8276-8279, 16180-16193, and 16258-16262</li>
<li><em>count</em> and <em>cost</em> search modes for neighbours</li>
<li>implementation of 28 block indels containing between 2 and 264 base-pairs (Table 1)</li>
</ol>
<table>
<thead>
<tr>
<th>No.</th>
<th>Insertion position</th>
<th>Deletion positions</th>
<th>Pattern</th>
<th>Length [in bp]</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>16032</td>
<td>16032.1-16032.15</td>
<td>TCTCTGTTCTTTCAT</td>
<td>15</td>
</tr>
<tr>
<td>2</td>
<td>16164</td>
<td>16165-16318</td>
<td>AACCCAATCCACA…</td>
<td>154</td>
</tr>
<tr>
<td>3</td>
<td>16309</td>
<td>16310-16316</td>
<td>GTACATA</td>
<td>7</td>
</tr>
<tr>
<td>4</td>
<td>104</td>
<td>105-110</td>
<td>CGGAGC</td>
<td>6</td>
</tr>
<tr>
<td>5</td>
<td>105</td>
<td>106-111</td>
<td>GGAGCA</td>
<td>6</td>
</tr>
<tr>
<td>6</td>
<td>209</td>
<td>209.1-209.7</td>
<td>GTGTGTT</td>
<td>7</td>
</tr>
<tr>
<td>7</td>
<td>241</td>
<td>241.1-241.3</td>
<td>TAA</td>
<td>3</td>
</tr>
<tr>
<td>8</td>
<td>286</td>
<td>286.1-286.5</td>
<td>TAACA</td>
<td>5</td>
</tr>
<tr>
<td>9</td>
<td>290</td>
<td>291-294</td>
<td>ATTT</td>
<td>4</td>
</tr>
<tr>
<td>10</td>
<td>291</td>
<td>291.1-291.16</td>
<td>ACATCATAACAAA…</td>
<td>16</td>
</tr>
<tr>
<td>11</td>
<td>292</td>
<td>292.1-292.2</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>12</td>
<td>292</td>
<td>292.2-292.4</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>13</td>
<td>306</td>
<td>307-356</td>
<td>CCCTCCCCCCGCT…</td>
<td>51</td>
</tr>
<tr>
<td>14</td>
<td>309</td>
<td>310-315</td>
<td>TCCCCC</td>
<td>6</td>
</tr>
<tr>
<td>15</td>
<td>315</td>
<td>316-319.0</td>
<td>GCTT</td>
<td>4</td>
</tr>
<tr>
<td>16</td>
<td>342</td>
<td>342.1-342.2</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>17</td>
<td>343</td>
<td>343.1-343.3</td>
<td>ATC</td>
<td>3</td>
</tr>
<tr>
<td>18</td>
<td>368</td>
<td>368.1-368.4</td>
<td>AGAA</td>
<td>4</td>
</tr>
<tr>
<td>19</td>
<td>398</td>
<td>398.1-398.14</td>
<td>ACCAGATTTCAAAT</td>
<td>14</td>
</tr>
<tr>
<td>20</td>
<td>470</td>
<td>471-478</td>
<td>TACTACTA</td>
<td>8</td>
</tr>
<tr>
<td>21</td>
<td>494</td>
<td>494-503</td>
<td>CCCCGCCCA</td>
<td>9</td>
</tr>
<tr>
<td>22</td>
<td>524</td>
<td>524.1-524.2</td>
<td>GC</td>
<td>2</td>
</tr>
<tr>
<td>23</td>
<td>563</td>
<td>563.1-563.204</td>
<td>AACAAAGAACCC...</td>
<td>204</td>
</tr>
<tr>
<td>24</td>
<td>588</td>
<td>568.1-588.16</td>
<td>CACAGTTTATGTA…</td>
<td>16</td>
</tr>
<tr>
<td>25</td>
<td>3326</td>
<td>3327-3590</td>
<td>ACTCCTCATTGTA…</td>
<td>264</td>
</tr>
<tr>
<td>26</td>
<td>6019</td>
<td>6020-6024</td>
<td>CGAGC</td>
<td>5</td>
</tr>
<tr>
<td>27</td>
<td>9486</td>
<td>9487-9501</td>
<td>TCGCAGGATTTTT…</td>
<td>15</td>
</tr>
<tr>
<td>28</td>
<td>14786</td>
<td>14787-14790</td>
<td>TTAA</td>
<td>4</td>
</tr>
<tr>
<td>29</td>
<td>16006</td>
<td>16006.1-16006.3</td>
<td>CTA</td>
<td>3</td>
</tr>
</tbody>
</table>
<h3 id="41-query-options">4.1 Query Options</h3>
<h4 id="411-sample-id">4.1.1. Sample ID</h4>
<p>Use this field to enter the ID of a mitotype. Search results are linked to this information and also provided on printouts. Sample IDs are used to identify queries in the search history of each individual user.</p>
<h4 id="412-ranges">4.1.2. Ranges</h4>
<p>Database queries require specification of the interpretation range(s) for rCRS-coded and FASTA-like string mitotypes. Typical ranges are: HVS-I (16024-16365), HVS-II (73-340), CR (16024-576), mitogenome (ALL).
Individual SNPs can be queried by indicating the SNP in the range field, e.g. 3010. Note that EMPOP depends on the sequence range provided by the submitting laboratory, which is why the ranges can be slightly different for some submissions/populations.</p>
<h4 id="413-mitotype">4.1.3. Mitotype</h4>
<p>Submit your mitotypes as FASTA-like sequence strings or reported relative to the rCRS. 
Query sequence strings:
Copy&amp;paste the sequence string from a text file or a consensus from sequence analysis software. Do not enter header information like in usual FASTA format; enter nucleotides only. For mixtures (e.g point heteroplasmy) use the extended IUPAC code (see below).</p>
<p>FIGURE 3</p>
<p><em>Query rCRS aligned mitotypes:</em>
Differences to the revised Cambridge Reference Sequence (rCRS, Andrews et al 1999) are entered as mitotypes.</p>
<table>
<thead>
<tr>
<th>Type</th>
<th>Possible annotations</th>
<th>Comment</th>
</tr>
</thead>
<tbody>
<tr>
<td>Base changes</td>
<td>73G, A73G</td>
<td>If preceding bases are used they must match rCRS base at the given   position</td>
</tr>
<tr>
<td>Insertions</td>
<td>315.1C     -315.1C     315+C     309.1C     309.2C     309+CC</td>
<td>For multiple insertions all preceding insertions need to be stated,   i.e. annotating 309.2C is not possible without annotating 309.1C</td>
</tr>
<tr>
<td>Deletions</td>
<td>249-     A249-     249delA     249del</td>
<td>'del' is treated case insensitive, e.g. Del, DEL, dEL, deL etc is   accepted.     Please note that the single character 'D' is considered a mixture of   A, G, and T (IUB code). The single character 'd' is considered a mixture of   A, G, T, and deletion (see Parson et al 2014 for details).</td>
</tr>
</tbody>
</table>
<p>::: warning
Note that the EMPOP query discerns capital letters (A,G, C, T, Y, …) from uncapitalized letters (a, g, c, t, y, …). Uncapizalized letters stand for a mixture of a deletion and a non-deleted variant. E.g. T152c represents two variants, T152C and T152del. 
:::</p>
<h4 id="414-release">4.1.4. Release</h4>
<p>EMPOP 4 offers release-specific queries. The most recent database release is selected by default. Earlier database releases can be selected if available. </p>
<h4 id="415-find-neighbors">4.1.5. Find neighbors</h4>
<p>EMPOP offers searching for neighbors by count and by cost. Under current settings EMPOP reports neighbors within a count of 2 differences or costs of 5.34 (see Huber et al 2018 for details).
Finding neighbors by count is the default setting for forensic frequency estimates.</p>
<h4 id="416-match-type">4.1.6. Match type</h4>
<p>This is relevant for the consideration of point heteroplasmy in both the query sequence as well as the database sequences.
<em>Pattern match:</em> mixture designations match its individual components (Y={C,T,Y}). Example: 152Y matches 152T and 152C.
<em>Literal match:</em> mixture designations are considered exclusive to all other nucleotide designations (Y={Y}). Example: 152Y matches only 152Y.
Pattern match is the default setting for forensic frequency estimates.</p>
<h4 id="417-disregard-indels">4.1.7. Disregard InDels</h4>
<p>Length variants that are known hotspots for insertion/deletions (indels) should be ignored in a forensic database query. This involves the C-runs around positions 16193, 309, 463 and 573 and the T-run around position 455 relative to the rCRS in the control region. In the coding region length variants around positions 960, 5899, 8276 and 8285 are ignored for a forensic query (Table 2).</p>
<table>
<thead>
<tr>
<th>Length variant</th>
<th>5’ junction</th>
<th>3’ junction</th>
<th>Repeating motif</th>
</tr>
</thead>
<tbody>
<tr>
<td>16193</td>
<td>16189</td>
<td>16194</td>
<td>C</td>
</tr>
<tr>
<td>309</td>
<td>302</td>
<td>316</td>
<td>C</td>
</tr>
<tr>
<td>315*</td>
<td>302</td>
<td>316</td>
<td>C</td>
</tr>
<tr>
<td>455</td>
<td>451</td>
<td>456</td>
<td>T</td>
</tr>
<tr>
<td>463</td>
<td>460</td>
<td>464</td>
<td>C</td>
</tr>
<tr>
<td>573</td>
<td>567</td>
<td>574</td>
<td>C</td>
</tr>
<tr>
<td>960</td>
<td>955</td>
<td>961</td>
<td>C</td>
</tr>
<tr>
<td>5899</td>
<td>5894</td>
<td>5900</td>
<td>C</td>
</tr>
<tr>
<td>8276</td>
<td>8271</td>
<td>8277</td>
<td>C</td>
</tr>
<tr>
<td>8285</td>
<td>8280</td>
<td>8286</td>
<td>C</td>
</tr>
</tbody>
</table>
<ul>
<li>Note that the C-insertion between 310 and 316 is a stable length variant, not a length hotspot. It is listed here for uninterrupted C-runs as a consequence of T310C.</li>
</ul>
<p>Standard query settings disregard discrepancies in hotspot length variant regions between query and database sequences.
Note that costs of disregarded InDels do not contribute to the final costs, which influences the ranking of results. See section 4.4. Neighbors.</p>
<h3 id="42-result">4.2. Result</h3>
<p>The execution of a database query automatically directs the user to the Results tab. Sample ID, query range(s) and mitotype are indicated in the top lines. Following information is listed in the results table:</p>
<ol>
<li>number of observed matches in the entire database</li>
<li>number of observed matches sorted by geographic origin and</li>
<li>number of observed matches by metapopulation affiliation</li>
</ol>
<p>FIGURE 4</p>
<p>An uncorrected frequency estimate is provided including a two-tailed Clopper Pearson confidence interval. Correction for sampling bias is provided and alternative methods to calculate probabilities are provided in the drop-down box to the right. P values can be estimated based on following formulas:</p>
<ol>
<li>(x+1)/(n+1)</li>
<li>(x+2)/(n+2)</li>
<li>CI from zero pop</li>
</ol>
<p>Where x… number of database hits and n… database size</p>
<p>Free text searches are possible for origin and metapopulation to address the relevant subset of the database. This depends on the formulation of the hypothesis, e.g. the reduction of the dataset to the country of Spain.</p>
<p>Note that the number of mitotypes included in a query result depends on the indicated sequence range. Only mitotypes with overlapping sequence ranges to the query sequence are considered. E.g. the query range 16024-576 includes all database sequences that were typed for the entire control region. HVS-I/II data (16024-16365 73-340) are not included in such a query. It may therefore be conservative to also perform a query with standard HVS-I/II sequence ranges.</p>
<p>Below the tabular representation of the database query an interactive map can be found that depicts the sampled populations within the query range (red) and the matches in the sampled populations (green).</p>
<p>FIGURE 5</p>
<h3 id="43-details">4.3. Details</h3>
<p>The Details tab provides a more detailed presentation of the matching mitotypes.</p>
<h4 id="431-when-matches-are-found">4.3.1. When matches are found</h4>
<p>FIGURE 6</p>
<p>EMPOP provides a summary table of all matching mitotypes that meet the queried sequence range. Columns can be sorted by clicking on the column headers.
Geographic and metapopulation origins can be filtered using the text boxes.
Ignored mutations list the differences between database and query sequences that were disregarded for the search (see 4.1.7. Disregard InDels ). The values in brackets display the costs of the listed mutation (details see Huber et al 2018).
Haplogroup indicates the samples’ haplogroup assignment. In case of a database match, there is no need to estimate the haplogroup as this column simply indicates the haplogroup of the matching samples. Rank 1 displays the haplogroup estimate with lowest costs (including a tolerance of 0.1) and Rank 2 displays the haplogroup estimate with the next lowest costs (including a tolerance of 0.1).</p>
<p>4.3.2. When no matches are found
MtDNA sequence queries often do not result in database matches. In these cases the Details tab stays empty.</p>
<p>EXAMPLE IMAGE</p>
<p>Unobserved mitotypes are indicated by a frequency value of “0”:</p>
<p>FIGURE 7</p>
<p>In Details no matches are listed:</p>
<p>FIGURE 8</p>
<h3 id="44-neighbors">4.4. Neighbors</h3>
<p>Similar sequences with a low number of differences are displayed here.</p>
<p>FIGURE 9</p>
<p>The display of neighbors follows the same concept as the summary of matches (see 4.3. Details) and includes all mitotypes that are at a distance to the query sequence of one and two differences (“events”).
An “Event” refers to the biological meaning of any difference but not the absolute number of differing nucleotides. As such, a tandem deletion (or insertion) in the AC-repeat region between 514 and 524 is regarded as one event, and therefore one difference between otherwise matching mitotypes. The same rationale applies to the 6 bp Chibcha deletion between 105 and 110 or 106 and 111, the 9 bp deletion between 8281 and 8290, as well as other (less abundant) block indels in the mitochondrial genome.
Additional information is provided with regard to differences between query mitotype and neighbors. These are listed in the columns cost, count and mutations.</p>
<p>Costs are determined by the change from the base profile symbol to the test profile symbol (approximately 1.0 for an average mutation). See Huber et al 2018 for further details.</p>
<p>Count lists the number of mutational events between query and database mitotypes.
Note that some combined mutations are single events, e.g. 523del 524del or 106-111del and treated as such in EMPOP.</p>
<p>Mutations specifies differences between query and database mitotypes, which are listed with the individual costs. Disregarded indels do not contribute to the final costs.</p>
<h3 id="45-alignment">4.5. Alignment</h3>
<p>Since EMPOP 4 the phylogenetic alignment is displayed as illustrated in Figure 10 below:</p>
<p>FIGURE 10</p>
<h3 id="46-haplogrouping">4.6. Haplogrouping</h3>
<p>The assignment of haplogroups to mitotypes contributes substantial value for quality control, not only in forensic genetics but also in population and medical genetics. The availability of Phylotree, a widely accepted phylogenetic tree of human mitochondrial DNA lineages, led to the development of several (semi-)automated software solutions for haplogrouping. However, the currently existing tools only make use of haplogroup-defining mutations, whereas private mutations (beyond the haplogroup level) can be additionally informative allowing for enhanced haplogroup assignment.</p>
<p>The scientific publication is currently under preparation, details will follow later. </p>
              
            </div>
          </div><footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="Footer Navigation">
        <a href="../register/" class="btn btn-neutral float-left" title="Register & Login"><span class="icon icon-circle-arrow-left"></span> Previous</a>
        <a href="../browsing_empop/" class="btn btn-neutral float-right" title="Browsing EMPOP for populations">Next <span class="icon icon-circle-arrow-right"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
      <span><a href="../register/" style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
      <span><a href="../browsing_empop/" style="color: #fcfcfc">Next &raquo;</a></span>
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
