<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>EMPOP Tools - EMPOP Documentation</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "EMPOP Tools";
        var mkdocs_page_input_path = "tools.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> EMPOP Documentation
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../revisions/">Revision Overview</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../introduction/">Introduction</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../concept/">Concept</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../register/">Register & Login</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../using_empop/">Using EMPOP for mitotype searches</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../browsing_empop/">Browsing EMPOP for populations</a>
                </li>
              </ul>
              <ul class="current">
                <li class="toctree-l1 current"><a class="reference internal current" href="#">EMPOP Tools</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#61-haplogroup-browser">6.1. Haplogroup Browser</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#62-empcheck">6.2. EMPcheck</a>
        <ul>
    <li class="toctree-l3"><a class="reference internal" href="#621-structure-of-the-emp-file">6.2.1. Structure of the emp-file</a>
    </li>
        </ul>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#63-network">6.3. NETWORK</a>
        <ul>
    <li class="toctree-l3"><a class="reference internal" href="#631-input">6.3.1. Input</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#632-output">6.3.2. Output</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#references-further-reading">References &amp; Further Reading</a>
    </li>
        </ul>
    </li>
    </ul>
                </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">EMPOP Documentation</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
      <li class="breadcrumb-item active">EMPOP Tools</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h2 id="6-empop-tools">6. EMPOP Tools</h2>
<p>The EMPOP tools section provides a suite of software to support the analysis and interpretation of mitochondrial DNA sequence variation.</p>
<h3 id="61-haplogroup-browser">6.1.    Haplogroup Browser</h3>
<p>This tool represents the established most recent Phylotree haplogroups in convenient searchable format and provides the number of EMPOP sequences assigned to the respective haplogroups by SAM2. Note that EMPOP provides the MRCA haplogroup if multiple haplogroup assignments are feasible. 
Individual haplogroups can also be found by querying differences to the rCRS in a database of &gt; 20.000 mtGenome sequences.</p>
<p>FIGURE 12</p>
<h3 id="62-empcheck">6.2.    EMPcheck</h3>
<p>EMPcheck is a tool to perform plausibility checks on an rCRS-coded data table.
The file format must meet the requirements described below and in Carracedo et al 2014.</p>
<h4 id="621-structure-of-the-emp-file">6.2.1. Structure of the emp-file</h4>
<p>Lines starting with "#!" indicate the sequence range of the mitotype. Note that a given sequence range is applied to all mtDNA mitotypes following this range until a new range is defined. Thus, multiple mitotypes with different sequence ranges can be handled in one file. 
The file lists the mitotypes in columns with the following contents.</p>
<ul>
<li>
<p>Column A: Sequence name: don't use blank space or special characters (allowed characters are letters (except umlauts ä, ö, ü), numbers, "-", "_", "/")</p>
</li>
<li>
<p>Column B: Haplogroup (hg) status: indicate hg, if unknown, use "?"</p>
</li>
<li>
<p>Column C: Frequency of mitotype (0 - 9999). Typically, this value is “1”, as individual mitotypes should be presented. If it is set to 0 the sample is not considered for the analysis.</p>
</li>
<li>
<p>Column D: Annotation of the mitotype relative to the rCRS. Separate differences by tabs (or use individual cells in MS Excel). Use forensic notation of sequences as outlined in the revised and updated ISFG recommendations for mtDNA typing (Parson et al (2014)).</p>
</li>
</ul>
<p>Text lines can be included everywhere in the file for comments or description. They need to be marked with "#".
Avoid blank lines (except when marked with "#").</p>
<p>The structure of an EMP file is illustrated below and the file can also be downloaded from: Downloads section in EMPOP.</p>
<p>EXAMPLE EMP FILE</p>
<h3 id="63-network">6.3.    NETWORK</h3>
<p>This tool can be used to calculate and draw quasi-median networks. They are useful to examine the quality of an mtDNA dataset.</p>
<p>MtDNA data tables can be depicted as quasi-median networks to enhance the understanding of the data in regard to homoplasmy and potential artifacts. Highly recurrent mutations are removed from the dataset (filtering) to help detect data idiosyncrasies that pinpoint sequencing and data interpretation problems. A detailed discussion of the method can be found in Bandelt and Dür (2007) and its application in Parson and Dür (2007). </p>
<p>The following section leads you through
- the input and parameter selection of a network analysis
- the output generated by NETWORK
- network drawing and
- interpretation of the results.</p>
<h4 id="631-input">6.3.1. Input</h4>
<p><strong>Sample Info:</strong>
The sample-specific information identifies a search. This is also the reference under which the query is reported. The history of NETWORK searches can be found under YOUR ACCOUNT.</p>
<p><strong>Input file (=emp file):</strong>
The input file contains the annotated population data. The emp-file is a tab delimited text file that can be created using standard text software or MS Excel (then, safe file under .txt format and rename "txt" by "emp"). Its format needs to meet the following criteria:</p>
<p><strong>Ambiguous symbols:</strong>
The software accepts the IUB-code. However, ambiguous symbols (e.g. sequence heteroplasmy Y ~ C/T) can cause artificial nodes and links in the network. Therefore it may be necessary to specify a non-ambiguous symbol either by calling the dominant type or by using the phylogenetic background of the sample. You will be notified on the presence of ambiguous symbols on the screen and in the network analysis report. For your information you also get a list of new insertions in your data set that are not known to the current EMPOP database.</p>
<p><strong>New insertions:</strong>
We collect positions with observed insertions in an EMPOP datafile to which new data are compared. New insertions that have not been recorded in EMPOP yet are displayed to draw the attention on them. This however does not impact the performance of NETWORK.</p>
<p><strong>Filtering:</strong>
Highly recurrent mutations are removed from the data set (filtering) that would otherwise increase the complexity of the network. You can choose between different filters depending on the application. The contents of the filters can be viewed by clicking on the symbol next to the dropdown box.</p>
<p><strong>Available filters:</strong>
- EMPOPspeedy: This filter removes highly recurrent mutations based on the lists provided in Bandelt et al (2002 and 2006). This filter is typically used for the analysis of mtDNA population data within the hypervariable segments - HVS-I (16024 - 16569) and HVS-II (1 - 576).
- EMPOPspeedyWE: This filter removes highly recurrent mutations as presented in Zimmermann et al (2010). This filter is typically used for the analysis of west Eurasian mtDNA population data within the hypervariable segments - HVS-I (16024 - 16569) and HVS-II (1 - 576).
- EMPOPall_R11: This is a superfine filter that contains all mutations observed in EMPOP. This filter provides a very quick check on the data by highlighting only yet unobserved mutations. We update the EMPOPall filter periodically.
- Unfiltered: None of the mutations are removed from your dataset. This is useful for the analysis of very short sequence stretches in the mtDNA CR (see below). The complexity of the network will increase rapidly if no filter is applied to the analysis of larger sequence regions.</p>
<p><strong>Range:</strong>
The range determines the region for which the network is computed. Any range within 16024-16569 and 1-576 can be queried. In some data very small regions may be interesting for detailed network analysis (e.g. 450-460).</p>
<p>Submit starts the execution.</p>
<h4 id="632-output">6.3.2. Output</h4>
<p>After clicking on the Submit Button, the network calculation is initiated. Depending on the size of the file and the used filter options this process may take some time.
When finished, result files will be listed in “Network Result Files” on the “your account” page.</p>
<p>FIGURE 13</p>
<p>Download the file and unzip it to obtain the folder [RID_FILTERNAME_REGION], which contains the following files:
- Results file [FILENAME_report.txt]: This file summarizes the settings and the results of the network analysis - for details see chapter Interpretation. 
- File for drawing the network [FILENAME_network.dnw]: This file can be used to draw the entire network of the mtDNA datafile by dnw.exe. 
- File for drawing the torso [FILENAME_torso.dnw]: This file can be used to draw the torso of the network of the mtDNA datafile by dnw.exe. 
- Difference table of the network [FILENAME_network.txt]: This file contains the filtered and reduced mitotypes of the entire network, displayed in dot table format. 
- Difference table of the torso [FILENAME_torso.txt]: This file contains the filtered and reduced mitotypes of the torso of the network, displayed in dot table format. 
- EMP-File [EMPFileName.emp]: The emp file which was uploaded
- Info-file [FILENAME_info.txt]: Contains the sample identification (which was defined in “Sample info”, see 6.3.1 Input) and the title of the emp file.</p>
<p><strong>Drawing</strong>
1.  Download the software for drawing the network (DrawNetWorkSetup.exe) from the EMPOP download page.
2.  Execute the file and follow the instructions given by the software. Choose a destination folder where the software is to be installed.
3.  Once the installation is finished you can find a folder called DrawNetWork containing the software and an uninstaller in the start menu. Files having ".dnw" as file ending are automatically linked to the software. Double-clicking a dnw file opens the network in a separate window. The help menu contains a legend of keys to edit the network (e.g. t ... for drawing a draft of the network, l ... for adding labels, etc.). During execution the current drawing can be exported in SVG (Scalable Vector Graphic), EPS (Encapsulated PostScript) or GIF format for printing or editing.</p>
<p><strong>Interpretation</strong>
The Report.txt file summarizes relevant information of the network analysis. The network is described in a table by the number of samples (n), the number of polymorphic positions (p), the number of partitions or condensed characters (p’), the number of mitotypes (h), the number of nodes in the network (q), the number of nodes in the torso (t) and the number of nodes of the peeled torso (t’). These values are indicative for the quality of a network. However, they depend on the size and composition of the population data set in question. Generally, small t’-values (ideally 1) describe a star-like structure of the network, which is in agreement with the expected evolutionary pattern. 
A more suggestive representation of the data is the graph of the quasi-median network. The nodes of this graph are given by the mitotypes or the quasi-medians generated from the mitotypes. In the drawing the frequencies of the mitotypes or quasi-medians are also shown. The root node is drawn with a bold circle and contains the filtered and reduced Anderson sequence (In the rare case that no mitotype contains the filtered and reduced Anderson sequence, the first mitotype is chosen instead and a warning is included in the report). The links are single or combined mutations specified by the syntax for single mutations or / for combined mutations, where the orientation is from the root node outwards. Links with the same mutation are drawn parallel and are labeled only once. The torso is obtained from the quasi-median network by collapsing all pendant subtrees into their base nodes. Thus the analysis of homoplasmy can be restricted to the torso which contains all the reticulation of the network. For each base node the coinciding mitotypes are listed in the report to make it easy to find all corresponding samples.</p>
<h4 id="references-further-reading">References &amp; Further Reading</h4>
<ul>
<li>Bandelt HJ et al (2002) The fingerprint of phantom mutations in mitochondrial DNA data. Am J Hum Genet 71:1150-1160</li>
<li>Bandelt HJ et al (2006) Estimation of mutation rates and coalescence times: some caveats. In: Human mitochondrial DNA and the evolution of Homo sapiens. Springer-Verlag eds. Hans-Jürgen Bandelt, Vincent Macaulay, Martin Richards</li>
<li>Bandelt and Dür (2007) Translating DNA data tables into quasi-median networks for parsimony analysis and error detection. Mol Phylogenet Evol 42:256-271</li>
<li>Parson and Dür (2007) EMPOP - A forensic mtDNA database. FSI:Genetics 1:88-92</li>
<li>Schwarz and Dür (2011) Visualization of quasi-median networks. Discrete Applied Mathematics 159(15):1608-1616</li>
<li>Zimmermann et al (2014) Improved visibility of character conflicts in quasi-median networks with the EMPOP NETWORK software. Croat Med J 55(2): 115-120.</li>
<li>Parson et al (2013) Evaluation of next generation mtGenome sequencing using the Ion Torrent Personal Genome Machine (PGM). FSI: Genetics 7(5): 543-549</li>
<li>Eduardoff et al (2017) Optimized mtDNA Control Region Primer Extension Capture Analysis for Forensically Relevant Samples and Highly Compromised mtDNA of Different Age and Origin. Genes 8(10)</li>
<li>Strobl et al (2018) Evaluation of the precision ID whole MtDNA genome panel for forensic analyses. FSI: Genetics 35: 21-25.</li>
</ul>
              
            </div>
          </div><footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="Footer Navigation">
        <a href="../browsing_empop/" class="btn btn-neutral float-left" title="Browsing EMPOP for populations"><span class="icon icon-circle-arrow-left"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
      <span><a href="../browsing_empop/" style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
