<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>EMPOP mtDNA Database - Directions for Use - EMPOP Documentation</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "EMPOP mtDNA Database - Directions for Use";
        var mkdocs_page_input_path = "empop_use.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> EMPOP Documentation
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../revisions/">Revision Overview</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../introduction/">Introduction</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../concept/">Concept</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../register/">Register & Login</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../using_empop/">Using EMPOP for mitotype searches</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../browsing_empop/">Browsing EMPOP for populations</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../tools/">EMPOP Tools</a>
                </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">EMPOP Documentation</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
      <li class="breadcrumb-item active">EMPOP mtDNA Database - Directions for Use</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h1 id="empop-mtdna-database-directions-for-use"><strong>EMPOP mtDNA Database - Directions for Use</strong></h1>
<h2 id="revision-overview"><strong>Revision Overview</strong></h2>
<p><strong>Version V4</strong></p>
<ul>
<li>
<p>April 2019</p>
<ul>
<li>added new block indel 494-503 (CCCCGCCCA)</li>
</ul>
</li>
<li>
<p>September 2018</p>
<ul>
<li>
<p>Release of EMPOP 4</p>
<ul>
<li>
<p>The new query engine SAM2 offers enhanced search functions,
    phylogenetic alignment and haplogrouping (see Huber et al
    2018 for details)</p>
</li>
<li>
<p>Revision of several sections of the USE document</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>December 2015</p>
<ul>
<li>
<p>Section 4.3.2. -- When no matches are found was updated</p>
<ul>
<li>Tolerance value of EMMA was changed from ,,0.3" to ,,0.1"</li>
</ul>
</li>
</ul>
</li>
<li>
<p>November 2015</p>
<ul>
<li>Section 4.1.8. EMPOP haplogroup estimation -- EMMA was added</li>
</ul>
</li>
<li>
<p>October 2015</p>
<ul>
<li>Information about special positions was added in Section 4.1.2.
    Ranges</li>
</ul>
</li>
<li>
<p>July 2015</p>
<ul>
<li>
<p>Revision Overview was added</p>
</li>
<li>
<p>Section 4.3.3. Ambiguous haplogroup estimates was added</p>
</li>
</ul>
</li>
<li>
<p>May2015</p>
<ul>
<li>Initial Release of EMPOP mtDNA Database -- Directions for Use</li>
</ul>
</li>
</ul>
<h2 id="1-introduction">1. Introduction</h2>
<p>The high copy number per cell, the stability against degradation and the
maternal mode of inheritance make the mitochondrial (mt) genome
particularly suitable for palaeo-, medical- and forensic genetic
investigations. Its increased evolutionary rate led to sequence
variation that has been generated by sequential accumulation of new
mutations along radiating maternal lineages during human dispersal into
different parts of the world.</p>
<p>Forensic molecular biology takes advantage of this variation for human
identity testing by sequence analysis of the (hypervariable segments
within the) mtDNA control region (CR). New developments in Massively
Parallel Sequencing (MPS) demonstrated that also full mtGenome
information can be obtained from even degraded forensic samples (Parson
et al 2013, Eduardoff et al 2017, Strobl et al 2018). MtDNA analysis is
a powerful tool to exclude samples as originating from the same
individual/matriline. If two samples cannot be excluded the significance
of the mtDNA match is assessed by making reference to the abundance of
that particular mtDNA sequence (= mitotype) in a relevant population.</p>
<h2 id="2-concept">2. Concept</h2>
<p>The EMPOP database aims at the collection, quality control and
searchable presentation of mtDNA mitotypes from all over the world.
EMPOP has carefully been envisioned and designed as high quality mtDNA
database, where available primary sequence lane data are permanently
linked to the database entries. The scientific concept and the quality
control measures using logical and phylogenetic tools were found
suitable for forensic purposes, e.g. by a declaration of the <a href="http://dl.empop.org/BGH-Urteil%20kombinierte%20DNA-Analyse.pdf">German
Supreme Court of Justice
(2010)</a>,
the <a href="http://swgdam.org/SWGDAM%20mtDNA_Interpretation_Guidelines_APPROVED_073013.pdf">SWGDAM mtDNA interpretation
guidelines</a>
(2013), and the <a href="http://www.ncbi.nlm.nih.gov/pubmed/25117402">updated ISFG guidelines for mtDNA
analysis</a> and
interpretation (2014).</p>
<p>The scientific contents presented in EMPOP were developed by the
<a href="http://www.gmi.eu">Institute of Legal Medicine (GMI)</a>, Medical
University of Innsbruck and the <a href="http://www.uibk.ac.at/mathematik/index.html.de">Institute of Mathematics, University of
Innsbruck</a>.</p>
<p>The mitotypes stored in EMPOP are not considered for partial or full
download. The concept of data quality management requires a centralized
supervision of the data. Necessary updates (e.g. haplogroup status,
Release updates) will be introduced by the database curators to ensure
continuous data quality and are made publicly available (see Release
history).</p>
<h2 id="3-registerlogin">3. Register/login</h2>
<p>An EMPOP user is identified by the Email address to which account
information (voluntary basis) and search history are connected. Follow
the instructions for registration. An Email will be sent with a link
that completes registration. Note that queries can be deleted in the
account history by the user.</p>
<p><img alt="Figure1 - User Registration" src="fig1_user_registration.png" /></p>
<h2 id="4-using-empop-for-mitotype-searches">4. Using EMPOP for mitotype searches</h2>
<p>EMPOP follows the <a href="http://www.ncbi.nlm.nih.gov/pubmed/25117402">revised and extended guidelines for mitochondrial DNA
typing</a> issued by the DNA
commission of the ISFG (Parson et al. 2014). See document for further
details.</p>
<p>[<strong>EMPOP's query engine - SAM 2:</strong>]{.underline}</p>
<p>EMPOP uses SAM 2, an updated and optimized software based on tests with
carefully curated full mitogenome sequences to perform unbiased and
conservative database queries to assist statistical evaluation of the
evidence in forensic practice. The major changes to the earlier version
of the software
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3064999/">SAM</a> (RÃ¶ck et al
2011) include:</p>
<p>i.  updated alignment/nomenclature conventions for the phylogenetically
    instable regions 50-70, 310-316, 455-460, 961-966, 8276-8279,
    16180-16193, and 16258-16262
ii. 'count' and 'cost' search modes for neighbours
iii. implementation of 28 block indels containing between 2 and 264
     base-pairs (Table 1)</p>
<table>
<thead>
<tr>
<th><strong>No.</strong></th>
<th><strong>Insertion Position</strong></th>
<th><strong>Deletion Positions</strong></th>
<th><strong>Pattern</strong></th>
<th><strong>Length (bp)</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>16032</td>
<td>16032.1-16032.15</td>
<td>TCTCTGTTCTTTCAT</td>
<td>15</td>
</tr>
<tr>
<td>2</td>
<td>16164</td>
<td>16165-16318</td>
<td>ACCCAATCCACA...</td>
<td>154</td>
</tr>
<tr>
<td>3</td>
<td>16309</td>
<td>16310-16316</td>
<td>GTACATA</td>
<td>7</td>
</tr>
<tr>
<td>4</td>
<td>104</td>
<td>105-110</td>
<td>CGGAGC</td>
<td>6</td>
</tr>
<tr>
<td>5</td>
<td>105</td>
<td>106-111</td>
<td>GGAGCA</td>
<td>6</td>
</tr>
<tr>
<td>6</td>
<td>209</td>
<td>209.1-209.7</td>
<td>GTGTGTT</td>
<td>7</td>
</tr>
<tr>
<td>7</td>
<td>241</td>
<td>241.1-241.3</td>
<td>TAA</td>
<td>3</td>
</tr>
<tr>
<td>8</td>
<td>286</td>
<td>286.1-286.5</td>
<td>TAACA</td>
<td>5</td>
</tr>
<tr>
<td>9</td>
<td>290</td>
<td>291-294</td>
<td>ATTT</td>
<td>4</td>
</tr>
<tr>
<td>10</td>
<td>291</td>
<td>291.1-291.16</td>
<td>ACATCATAACAAA...</td>
<td>16</td>
</tr>
<tr>
<td>11</td>
<td>292</td>
<td>292.1-292.2</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>12</td>
<td>292</td>
<td>292.2-292.4</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>13</td>
<td>306</td>
<td>307-356</td>
<td>CCTCCCCTCGCT...</td>
<td>51</td>
</tr>
<tr>
<td>14</td>
<td>309</td>
<td>310-315</td>
<td>TCCCCC</td>
<td>6</td>
</tr>
<tr>
<td>15</td>
<td>315</td>
<td>316-319.0</td>
<td>GCTT</td>
<td>4</td>
</tr>
<tr>
<td>16</td>
<td>342</td>
<td>342.1-342.2</td>
<td>AT</td>
<td>2</td>
</tr>
<tr>
<td>17</td>
<td>343</td>
<td>343.1-343.3</td>
<td>ATC</td>
<td>3</td>
</tr>
<tr>
<td>18</td>
<td>368</td>
<td>368.1-368.4</td>
<td>AGAA</td>
<td>4</td>
</tr>
<tr>
<td>19</td>
<td>398</td>
<td>398.1-398.14</td>
<td>ACCAGATTTCAAAT</td>
<td>14</td>
</tr>
<tr>
<td>20</td>
<td>470</td>
<td>471-478</td>
<td>TACTACTA</td>
<td>8</td>
</tr>
<tr>
<td>21</td>
<td>494</td>
<td>494-503</td>
<td>CCCGCCCA</td>
<td>9</td>
</tr>
<tr>
<td>22</td>
<td>524</td>
<td>524.1-524.2</td>
<td>GC</td>
<td>2</td>
</tr>
<tr>
<td>23</td>
<td>563</td>
<td>563.1-563.204</td>
<td>ACAAAAGAA</td>
<td>204</td>
</tr>
<tr>
<td>24</td>
<td>588</td>
<td>588.1-588.16</td>
<td>ACAGTTTA</td>
<td>16</td>
</tr>
<tr>
<td>25</td>
<td>3326</td>
<td>3327-3590</td>
<td>CTCCTCATGTA...</td>
<td>264</td>
</tr>
<tr>
<td>26</td>
<td>6019</td>
<td>6020-6024</td>
<td>CGAGC</td>
<td>5</td>
</tr>
<tr>
<td>27</td>
<td>9486</td>
<td>9487-9501</td>
<td>TGCAGGATTTTT...</td>
<td>15</td>
</tr>
<tr>
<td>28</td>
<td>14786</td>
<td>14787-14790</td>
<td>TTAA</td>
<td>4</td>
</tr>
<tr>
<td>29</td>
<td>16006</td>
<td>16006.1-16006.3</td>
<td>CTA</td>
<td>3</td>
</tr>
</tbody>
</table>
<h2 id="41-query-options">4.1. Query options</h2>
<p><img alt="" src="fig2_query_input.png" /></p>
<h3 id="411-sample-id">4.1.1. Sample ID</h3>
<p>Use this field to enter the ID of a mitotype. Search results are linked
to this information and also provided on printouts. Sample IDs are used
to identify queries in the search history of each individual user.</p>
<h3 id="412-ranges">4.1.2. Ranges</h3>
<p>Database queries require specification of the interpretation range(s)
for rCRS-coded and FASTA-like string mitotypes. Typical ranges are:
HVS-I (16024-16365), HVS-II (73-340), CR (16024-576), mitogenome (ALL).
Individual SNPs can be queried by indicating the SNP in the range field,
e.g. 3010. Note that EMPOP depends on the sequence range provided by the
submitting laboratory, which is why the ranges can be slightly different
for some submissions/populations.</p>
<p><strong>Examples:</strong></p>
<p>+------------+---------------------------------------------------------+
| 1          | represents a standard range for a query in HVS-I and    |
| 6024-16356 | HVS-II.                                                 |
| 73-340     |                                                         |
+------------+---------------------------------------------------------+
| 16024-576  | represents the control region range                     |
+------------+---------------------------------------------------------+
| 1          | represents a query range including HVS-I and the two    |
| 6024-16365 | SNPs 489 and 3010. Note that an insertion between 489   |
| 489 3010   | and 490 would not be included in that query range.      |
+------------+---------------------------------------------------------+
| ALL        | represents the complete mitogenome.                     |
+------------+---------------------------------------------------------+</p>
<p><strong>Note that there are special positions where a query does not make
sense. For example, position 3107: A deletion at this position cannot be
queried as 3107 serves as place holder in the rCRS to keep the original
numbering system downstream of that position (the CRS mistakenly
included that position).</strong></p>
<h3 id="413-mitotype">4.1.3. Mitotype</h3>
<p>Submit your mitotypes as FASTA-like sequence strings or reported
relative to the rCRS.</p>
<p>[Query sequence strings:]{.underline}</p>
<p>Copy&amp;paste the sequence string from a text file or a consensus from
sequence analysis software. Do not enter header information like in
usual FASTA format; enter nucleotides only. For mixtures (e.g point
heteroplasmy) use the extended <a href="https://www.bioinformatics.org/sms/iupac.html">IUPAC
code</a> (see below).</p>
<p><img alt="" src="fig3_fasta_input.png" /></p>
<p>[\
Query rCRS aligned mitotypes:]{.underline}</p>
<p>Differences to the revised Cambridge Reference Sequence (rCRS, <a href="http://www.ncbi.nlm.nih.gov/pubmed/10508508">Andrews
et al 1999</a>) are entered as
mitotypes.</p>
<p>Table 2 - Notation Guidelines</p>
<p>+---------------+------------------------+------------------------+
| <strong>Type</strong>      | <strong>Possible             | </strong>Comment<strong>            |
|               | annotations</strong>          |                        |
+:=============:+:======================:+:======================:+
| Base changes  | 73G, A73G              | If preceding bases are |
|               |                        | used they must match   |
|               |                        | rCRS base at the given |
|               |                        | position               |
+---------------+------------------------+------------------------+
| Insertions    | 315.1C                 | For multiple           |
|               |                        | insertions all         |
|               | -315.1C                | preceding insertions   |
|               |                        | need to be stated,     |
|               | 315+C                  | i.e. annotating 309.2C |
|               |                        | is not possible        |
|               | 309.1C                 | without annotating     |
|               |                        | 309.1C                 |
|               | 309.2C                 |                        |
|               |                        |                        |
|               | 309+CC                 |                        |
+---------------+------------------------+------------------------+
| Deletions     | 249-                   | 'del' is treated case  |
|               |                        | insensitive, e.g. Del, |
|               | A249-                  | DEL, dEL, deL etc is   |
|               |                        | accepted.              |
|               | 249delA                |                        |
|               |                        | Please note that the   |
|               | 249del                 | single character 'D'   |
|               |                        | is considered a        |
|               |                        | mixture of A, G, and T |
|               |                        | (IUB code). The single |
|               |                        | character 'd' is       |
|               |                        | considered a mixture   |
|               |                        | of A, G, T, and        |
|               |                        | deletion (see Parson   |
|               |                        | et al. 2014 for        |
|               |                        | details).              |
+---------------+------------------------+------------------------+</p>
<p><strong>Note that the EMPOP query discerns capital letters (A,G, C, T, Y, ...)
from uncapitalized letters (a, g, c, t, y, ...). Uncapizalized letters
stand for a mixture of a deletion and a non-deleted variant. E.g. T152c
represents two variants, T152C and T152del.</strong></p>
<h3 id="414-release">4.1.4. Release</h3>
<p>EMPOP 4 offers release-specific queries. The most recent database
release is selected by default. Earlier database releases can be
selected if available.</p>
<h3 id="415-find-neighbors">4.1.5. Find neighbors</h3>
<p>EMPOP offers searching for neighbors by count and by cost. Under current
settings EMPOP reports neighbors within a count of 2 differences or
costs of 5.34 (see Huber et al 2018 for details).</p>
<p>Finding neighbors by count is the default setting for forensic frequency
estimates.</p>
<h3 id="416-match-type">4.1.6. Match type</h3>
<p>This is relevant for the consideration of point heteroplasmy in both the
query sequence as well as the database sequences.</p>
<p>[Pattern match:]{.underline} mixture designations match its individual
components (Y={C,T,Y}). Example: 152Y matches 152T and 152C.</p>
<p>[Literal match:]{.underline} mixture designations are considered
exclusive to all other nucleotide designations (Y={Y}). Example: 152Y
matches only 152Y.</p>
<p>Pattern match is the default setting for forensic frequency estimates.</p>
<h3 id="417-disregard-indels">4.1.7. Disregard InDels</h3>
<p>Length variants that are known hotspots for insertion/deletions (indels)
should be ignored in a forensic database query. This involves the C-runs
around positions 16193, 309, 463 and 573 and the T-run around position
455 relative to the rCRS in the control region. In the coding region
length variants around positions 960, 5899, 8276 and 8285 are ignored
for a forensic query (Table 2).</p>
<p>Table 3 - List of implemented length hotspots:</p>
<p>+:---------------:+:--------------:+:--------------:+:---------------:+
| <strong>Length v      | </strong>5' j u       | <strong>3' j u       | </strong>R epeating    |
| ariant<strong>        | nction</strong>       | nction<strong>       | motif</strong>         |
+-----------------+----------------+----------------+-----------------+
| 16193           | 16189          | 16194          | C               |
+-----------------+----------------+----------------+-----------------+
| 309             | 302            | 316            | C               |
+-----------------+----------------+----------------+-----------------+
| 315*           | 302            | 316            | C               |
+-----------------+----------------+----------------+-----------------+
| 455             | 451            | 456            | T               |
+-----------------+----------------+----------------+-----------------+
| 463             | 460            | 464            | C               |
+-----------------+----------------+----------------+-----------------+
| 573             | 567            | 574            | C               |
+-----------------+----------------+----------------+-----------------+
| 960             | 955            | 961            | C               |
+-----------------+----------------+----------------+-----------------+
| 5899            | 5894           | 5900           | C               |
+-----------------+----------------+----------------+-----------------+
| 8276            | 8271           | 8277           | C               |
+-----------------+----------------+----------------+-----------------+
| 8285            | 8280           | 8286           | C               |
+-----------------+----------------+----------------+-----------------+</p>
<p>* Note that the C-insertion between 310 and 316 is a stable length
variant, not a length hotspot. It is listed here for uninterrupted
C-runs as a consequence of T310C.</p>
<p>Standard query settings disregard discrepancies in hotspot length
variant regions between query and database sequences.</p>
<p><strong>Note that costs of disregarded InDels do not contribute to the final
costs, which influences the ranking of results. See section 4.4.
Neighbors.</strong></p>
<h2 id="42-result">4.2. Result</h2>
<p>The execution of a database query automatically directs the user to the
<strong>Results</strong> tab. Sample ID, query range(s) and mitotype are indicated in
the top lines. Following information is listed in the results table:</p>
<ol>
<li>
<p>number of observed matches in the entire database</p>
</li>
<li>
<p>number of observed matches sorted by geographic origin and</p>
</li>
<li>
<p>number of observed matches by metapopulation affiliation</p>
</li>
</ol>
<p><img alt="" src="fig4_query_result.png" /></p>
<p>An uncorrected frequency estimate is provided including a two-tailed
Clopper Pearson confidence interval. Correction for sampling bias is
provided and alternative methods to calculate probabilities are provided
in the drop-down box to the right. P values can be estimated based on
following formulas:</p>
<ol>
<li>
<p>(x+1)/(n+1)</p>
</li>
<li>
<p>(x+2)/(n+2)</p>
</li>
<li>
<p>CI from zero pop</p>
</li>
</ol>
<p><em>Where x... number of database hits and n... database size</em></p>
<p>Free text searches are possible for origin and metapopulation to address
the relevant subset of the database. This depends on the formulation of
the hypothesis, e.g. the reduction of the dataset to the country of
Spain.</p>
<p><strong>Note that the number of mitotypes included in a query result depends
on the indicated sequence range. Only mitotypes with overlapping
sequence ranges to the query sequence are considered. E.g. the query
range 16024-576 includes all database sequences that were typed for the
entire control region. HVS-I/II data (16024-16365 73-340) are not
included in such a query. It may therefore be conservative to also
perform a query with standard HVS-I/II sequence ranges.</strong></p>
<p>Below the tabular representation of the database query an interactive
map can be found that depicts the sampled populations within the query
range (red) and the matches in the sampled populations (green).</p>
<p><img alt="" src="fig5_map.png" /></p>
<h2 id="43-details">4.3. Details</h2>
<p>The Details tab provides a more detailed presentation of the matching
mitotypes.</p>
<h3 id="431-when-matches-are-found">4.3.1. When matches are found</h3>
<p><img alt="" src="fig6_details.png" /></p>
<p>EMPOP provides a summary table of all matching mitotypes that meet the
queried sequence range. Columns can be sorted by clicking on the column
headers.</p>
<p><strong>Geographic</strong> and <strong>metapopulation</strong> origins can be filtered using the
text boxes.</p>
<p><strong>Ignored mutations</strong> list the differences between database and query
sequences that were disregarded for the search (see 4.1.7. Disregard
InDels ). The values in brackets display the costs of the listed
mutation (details see Huber et al 2018).</p>
<p><strong>Haplogroup</strong> indicates the samples' haplogroup assignment. In case of
a database match, there is no need to estimate the haplogroup as this
column simply indicates the haplogroup of the matching samples. Rank 1
displays the haplogroup estimate with lowest costs (including a
tolerance of 0.1) and Rank 2 displays the haplogroup estimate with the
next lowest costs (including a tolerance of 0.1).</p>
<h3 id="432-when-no-matches-are-found">4.3.2. When no matches are found</h3>
<p>MtDNA sequence queries often do not result in database matches. In these
cases the Details tab stays empty.</p>
<p><strong>Example:</strong></p>
<p><img alt="" src="4_3_2_not_matches.png" /></p>
<p>Unobserved mitotypes are indicated by a frequency value of "0":</p>
<p><img alt="" src="fig7_no_matches.png" /></p>
<p>In Details no matches are listed:</p>
<p><img alt="" src="fig8_no_matches.png" /></p>
<h2 id="44-neighbors">4.4. Neighbors</h2>
<p>Similar sequences with a low number of differences are displayed here.</p>
<p><img alt="" src="fig9_neighbours.png" /></p>
<p>The display of neighbors follows the same concept as the summary of
matches (see 4.3. Details) and includes all mitotypes that are at a
distance to the query sequence of <strong>one and two differences
("events")</strong>.</p>
<p>An "Event" refers to the biological meaning of any difference but not
the absolute number of differing nucleotides. As such, a tandem deletion
(or insertion) in the AC-repeat region between 514 and 524 is regarded
as one event, and therefore one difference between otherwise matching
mitotypes. The same rationale applies to the 6 bp Chibcha deletion
between 105 and 110 or 106 and 111, the 9 bp deletion between 8281 and
8290, as well as other (less abundant) block indels in the mitochondrial
genome.</p>
<p>Additional information is provided with regard to differences between
query mitotype and neighbors. These are listed in the columns <strong>cost</strong>,
<strong>count</strong> and <strong>mutations</strong>.</p>
<p><strong>Costs</strong> are determined by the change from the base profile symbol to
the test profile symbol (approximately 1.0 for an average mutation). See
Huber et al 2018 for further details.</p>
<p><strong>Count</strong> lists the number of mutational events between query and
database mitotypes.</p>
<blockquote>
<p><strong>Note that some combined mutations are single events, e.g. 523del
524del or 106-111del and treated as such in EMPOP.</strong></p>
</blockquote>
<p><strong>Mutations</strong> specifies differences between query and database
mitotypes, which are listed with the individual costs. Disregarded
indels do not contribute to the final costs.</p>
<h2 id="45-alignment">4.5. Alignment</h2>
<p>Since EMPOP 4 the phylogenetic alignment is displayed as illustrated in
Figure 10 below:</p>
<p><img alt="Figure 10 - Phylogenetic Alignment" src="fig10_alignment.png" /></p>
<h2 id="46-haplogrouping">4.6. Haplogrouping</h2>
<p>The assignment of haplogroups to mitotypes contributes substantial value
for quality control, not only in forensic genetics but also in
population and medical genetics. The availability of Phylotree, a widely
accepted phylogenetic tree of human mitochondrial DNA lineages, led to
the development of several (semi-)automated software solutions for
haplogrouping. However, the currently existing tools only make use of
haplogroup-defining mutations, whereas private mutations (beyond the
haplogroup level) can be additionally informative allowing for enhanced
haplogroup assignment.</p>
<p>The scientific publication is currently under preparation, details will
follow later.</p>
<h1 id="5-browsing-empop-for-populations">5. Browsing EMPOP for populations</h1>
<p><img alt="" src="fig11_populations.png" /></p>
<p>Under the tab POPULATIONS the individual datasets contained in EMPOP can
be found by using the accession number (if known), geographic or
metapopulation affiliations. Published datasets can be searched by Text
(Title) and Authors.</p>
<h2 id="6-empop-tools">6. EMPOP Tools</h2>
<p>The EMPOP tools section provides a suite of software to support the
analysis and interpretation of mitochondrial DNA sequence variation.</p>
<h3 id="61haplogroup-browser">6.1.Haplogroup Browser</h3>
<p>This tool represents the established most recent
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18853457">Phylotree</a> haplogroups in
convenient searchable format and provides the number of EMPOP sequences
assigned to the respective haplogroups by SAM2. Note that EMPOP provides
the MRCA haplogroup if multiple haplogroup assignments are feasible.</p>
<p>Individual haplogroups can also be found by querying differences to the
rCRS in a database of &gt; 20.000 mtGenome sequences.</p>
<p><img alt="" src="fig12_populations.png" /></p>
<h3 id="62empcheck">6.2.EMPcheck</h3>
<p>EMPcheck is a tool to perform plausibility checks on an rCRS-coded data
table.</p>
<p>The file format must meet the requirements described below and in
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24503419">Carracedo et al 2014</a>.</p>
<p><strong>Structure of the emp-file</strong></p>
<p>Lines starting with "#!" indicate the sequence range of the mitotype.
Note that a given sequence range is applied to all mtDNA mitotypes
following this range until a new range is defined. Thus, multiple
mitotypes with different sequence ranges can be handled in one file.</p>
<p>The file lists the mitotypes in columns with the following contents.</p>
<p>Column A: Sequence name: don't use blank space or special characters
(allowed characters are letters (except umlauts Ã¤, Ã¶, Ã¼), numbers, "-",
"_", "/")</p>
<p>Column B: Haplogroup (hg) status: indicate hg, if unknown, use "?"</p>
<p>Column C: Frequency of mitotype (0 - 9999). Typically, this value is
"1", as individual mitotypes should be presented. If it is set to 0 the
sample is not considered for the analysis.</p>
<p>Column D: Annotation of the mitotype relative to the rCRS. Separate
differences by tabs (or use individual cells in MS Excel). Use forensic
notation of sequences as outlined in the revised and updated ISFG
recommendations for mtDNA typing (<a href="http://www.ncbi.nlm.nih.gov/pubmed/25117402">Parson et al
(2014)</a>).</p>
<p>Text lines can be included everywhere in the file for comments or
description. They need to be marked with "#".</p>
<p>Avoid blank lines (except when marked with "#").</p>
<p>The structure of an EMP file is illustrated below and the file can also
be downloaded from: <a href="http://empop.org/downloads">Downloads</a> section in
EMPOP.</p>
<p><strong>Example</strong></p>
<p>| <span style = "color:green">
| # Population data of 250 individuals from Austria; Walther Parson (<a href="mailto:walther.parson@i-med.ac.at">walther.parson\@i-med.ac.at</a>{.email})
| # 100 samples from Innsbruck, 100 samples from Salzburg, 50 samples from Vienna</span>
| [#! 16024-576]{style="color:red"}
| mitotype1 H1c 1 16519C 263G 523DEL 524DEL 477C
| mitotype2 R0 2
| [#! 16024-16365 73-340]{style="color:red"}
| mitotype4 T2b 1 16126C 16294T 16296T 16304C 73G 263G 315.1C
| mitotype5 ? 1 16223T 73G 263G 315.1C</p>
<h3 id="63-network">6.3. NETWORK</h3>
<p>This tool can be used to calculate and draw quasi-median networks. They
are useful to examine the quality of an mtDNA dataset.</p>
<p>MtDNA data tables can be depicted as quasi-median networks to enhance
the understanding of the data in regard to homoplasmy and potential
artifacts. Highly recurrent mutations are removed from the dataset
(filtering) to help detect data idiosyncrasies that pinpoint sequencing
and data interpretation problems. A detailed discussion of the method
can be found in <a href="http://www.ncbi.nlm.nih.gov/pubmed/16945554">Bandelt and DÃ¼r
(2007)</a> and its application
in <a href="http://www.ncbi.nlm.nih.gov/pubmed/19083735">Parson and DÃ¼r (2007)</a>.</p>
<p>The following section leads you through</p>
<ul>
<li>
<p>the input and parameter selection of a network analysis</p>
</li>
<li>
<p>the output generated by NETWORK</p>
</li>
<li>
<p>network drawing and</p>
</li>
<li>
<p>interpretation of the results.</p>
</li>
</ul>
<p><strong>Input</strong></p>
<p><strong>Sample Info</strong></p>
<p>The sample-specific information identifies a search. This is also the
reference under which the query is reported. The history of NETWORK
searches can be found under YOUR ACCOUNT.</p>
<p><strong>Input file (=emp file)</strong></p>
<p>The input file contains the annotated population data. The emp-file is a
tab delimited text file that can be created using standard text software
or MS Excel (then, safe file under .txt format and rename "txt" by
"emp"). Its format needs to meet the following criteria:</p>
<p><strong>Ambiguous symbols</strong></p>
<p>The software accepts the IUB-code. However, ambiguous symbols (e.g.
sequence heteroplasmy Y \~ C/T) can cause artificial nodes and links in
the network. Therefore it may be necessary to specify a non-ambiguous
symbol either by calling the dominant type or by using the phylogenetic
background of the sample. You will be notified on the presence of
ambiguous symbols on the screen and in the network analysis report. For
your information you also get a list of new insertions in your data set
that are not known to the current EMPOP database.</p>
<p><strong>New insertions</strong></p>
<p>We collect positions with observed insertions in an EMPOP datafile to
which new data are compared. New insertions that have not been recorded
in EMPOP yet are displayed to draw the attention on them. This however
does not impact the performance of NETWORK.</p>
<p><strong>Filtering</strong></p>
<p>Highly recurrent mutations are removed from the data set (filtering)
that would otherwise increase the complexity of the network. You can
choose between different filters depending on the application. The
contents of the filters can be viewed by clicking on the symbol next to
the dropdown box.</p>
<p><strong>Available filters:</strong></p>
<ul>
<li>
<p>EMPOPspeedy: This filter removes highly recurrent mutations based on
    the lists provided in Bandelt et al (2002 and 2006). This filter is
    typically used for the analysis of mtDNA population data within the
    hypervariable segments - HVS-I (16024 - 16569) and HVS-II (1 - 576).</p>
</li>
<li>
<p>EMPOPspeedyWE: This filter removes highly recurrent mutations as
    presented in [Zimmermann et al (2010)]{.underline}. This filter is
    typically used for the analysis of west Eurasian mtDNA population
    data within the hypervariable segments - HVS-I (16024 - 16569) and
    HVS-II (1 - 576).</p>
</li>
<li>
<p>EMPOPall_R11: This is a superfine filter that contains all mutations
    observed in EMPOP. This filter provides a very quick check on the
    data by highlighting only yet unobserved mutations. We update the
    EMPOPall filter periodically.</p>
</li>
<li>
<p>Unfiltered: None of the mutations are removed from your dataset.
    This is useful for the analysis of very short sequence stretches in
    the mtDNA CR (see below). The complexity of the network will
    increase rapidly if no filter is applied to the analysis of larger
    sequence regions.</p>
</li>
</ul>
<p><strong>Range</strong></p>
<p>The range determines the region for which the network is computed. Any
range within 16024-16569 and 1-576 can be queried. In some data very
small regions may be interesting for detailed network analysis (e.g.
450-460).</p>
<p><strong>Submit</strong> starts the execution.</p>
<p><strong>Output</strong></p>
<p>After clicking on the <strong>Submit</strong> Button, the network calculation is
initiated. Depending on the size of the file and the used filter options
this process may take some time. When finished, result files will be
listed in "Network Result Files" on the "your account" page.</p>
<p><img alt="" src="fig13_network.png" /></p>
<p>Download the file and unzip it to obtain the folder
"RID_FILTERNAME_REGION", which contains the following files:</p>
<p><strong>Results file</strong>\
This file summarizes the settings and the results of the network
analysis - for details see chapter Interpretation.</p>
<p><strong>File for drawing the network\
</strong>This file can be used to draw the entire network of the mtDNA datafile
by dnw.exe.</p>
<p><strong>File for drawing the torso</strong>\
This file can be used to draw the torso of the network of the mtDNA
datafile by dnw.exe.</p>
<p><strong>Difference table of the network</strong>\
This file contains the filtered and reduced mitotypes of the entire
network, displayed in dot table format.</p>
<p><strong>Difference table of the torso</strong>\
This file contains the filtered and reduced mitotypes of the torso of
the network, displayed in dot table format.</p>
<p><strong>EMP-File</strong>\
The emp file which was uploaded</p>
<p><strong>Info-file</strong>\
Contains the sample identification (which was defined in "Sample info",
see 6.3.1 Input) and the title of the emp file.</p>
<p><strong>Drawing</strong></p>
<ol>
<li>
<p>Download the software for drawing the network
    (<a href="http://empop.org/modules/downloads/0/0/DrawNetWorkSetup.exe">DrawNetWorkSetup.exe</a>)
    from the EMPOP <a href="http://empop.org/downloads">download page</a>.</p>
</li>
<li>
<p>Execute the file and follow the instructions given by the software.
    Choose a destination folder where the software is to be installed.</p>
</li>
<li>
<p>Once the installation is finished you can find a folder called
    DrawNetWork containing the software and an uninstaller in the start
    menu. Files having ".dnw" as file ending are automatically linked to
    the software. Double-clicking a dnw file opens the network in a
    separate window. The help menu contains a legend of keys to edit the
    network (e.g. t ... for drawing a draft of the network, l ... for
    adding labels, etc.). During execution the current drawing can be
    exported in SVG (Scalable Vector Graphic), EPS (Encapsulated
    PostScript) or GIF format for printing or editing.</p>
</li>
</ol>
<p><strong>Interpretation</strong></p>
<p>The Report.txt file summarizes relevant information of the network
analysis. The network is described in a table by the number of samples
(n), the number of polymorphic positions (p), the number of partitions
or condensed characters (p'), the number of mitotypes (h), the number of
nodes in the network (q), the number of nodes in the torso (t) and the
number of nodes of the peeled torso (t'). These values are indicative
for the quality of a network. However, they depend on the size and
composition of the population data set in question. Generally, small
t'-values (ideally 1) describe a star-like structure of the network,
which is in agreement with the expected evolutionary pattern.</p>
<p>A more suggestive representation of the data is the graph of the
quasi-median network. The nodes of this graph are given by the mitotypes
or the quasi-medians generated from the mitotypes. In the drawing the
frequencies of the mitotypes or quasi-medians are also shown. The root
node is drawn with a bold circle and contains the filtered and reduced
Anderson sequence (In the rare case that no mitotype contains the
filtered and reduced Anderson sequence, the first mitotype is chosen
instead and a warning is included in the report). The links are single
or combined mutations specified by the syntax for single mutations or /
for combined mutations, where the orientation is from the root node
outwards. Links with the same mutation are drawn parallel and are
labeled only once. The torso is obtained from the quasi-median network
by collapsing all pendant subtrees into their base nodes. Thus the
analysis of homoplasmy can be restricted to the torso which contains all
the reticulation of the network. For each base node the coinciding
mitotypes are listed in the report to make it easy to find all
corresponding samples.</p>
<p><strong>References &amp; Further Reading</strong></p>
<ul>
<li>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/12384858">Bandelt HJ et al
    (2002)</a> The fingerprint
    of phantom mutations in mitochondrial DNA data. Am J Hum Genet
    71:1150-1160</p>
</li>
<li>
<p><a href="http://link.springer.com/chapter/10.1007%2F3-540-31789-9_4">Bandelt HJ et al
    (2006)</a>
    Estimation of mutation rates and coalescence times: some caveats.
    In: Human mitochondrial DNA and the evolution of Homo sapiens.
    Springer-Verlag eds. Hans-JÃ¼rgen Bandelt, Vincent Macaulay, Martin
    Richards</p>
</li>
<li>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/16945554">Bandelt and DÃ¼r
    (2007)</a> Translating DNA
    data tables into quasi-median networks for parsimony analysis and
    error detection. Mol Phylogenet Evol 42:256-271</p>
</li>
<li>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/19083735">Parson and DÃ¼r (2007)</a>
    EMPOP - A forensic mtDNA database. FSI:Genetics 1:88-92</p>
</li>
<li>
<p><a href="http://dl.acm.org/citation.cfm?id=2016534">Schwarz and DÃ¼r (2011)</a>
    Visualization of quasi-median networks. Discrete Applied Mathematics
    159(15):1608-1616</p>
</li>
<li>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4020147/">Zimmermann et al
    (2014)</a>
    Improved visibility of character conflicts in quasi-median networks
    with the EMPOP NETWORK software. Croat Med J 55(2): 115-120.</p>
</li>
<li>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/23948325">Parson et al (2013)</a>
    Evaluation of next generation mtGenome sequencing using the Ion
    Torrent Personal Genome Machine (PGM). FSI: Genetics 7(5): 543-549</p>
</li>
<li>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/28934125">Eduardoff et al
    (2017)</a> Optimized
    mtDNA Control Region Primer Extension Capture Analysis for
    Forensically Relevant Samples and Highly Compromised mtDNA of
    Different Age and Origin. Genes 8(10)</p>
</li>
<li>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/29626805">Strobl et al (2018)</a>
    Evaluation of the precision ID whole MtDNA genome panel for forensic
    analyses. FSI: Genetics 35: 21-25.</p>
</li>
</ul>
              
            </div>
          </div><footer>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
